u-root.go

```diff
@@ -83,6 +100,10 @@ func Main() error {
 	if env.GOOS != "linux" {
 		log.Printf("GOOS is not linux. Did you mean to set GOOS=linux?")
 	}
+	if !isRecommendedVersion() {
+		log.Printf("WARNING: You are not using one of the recommended Go versions (recommended = %v, have = %s). Some packages may not compile.",
```

Comment:
Add link to the golang website's instructions on updating Go.

Reply:
I don't need to google that for them. It's on the Go website, and I know nothing about their environment or the right way to install Go for them. (package manager? compile from source? use the dl/go1.12 download package?)